csunplugged/docker-development-entrypoint.sh

```diff
@@ -19,6 +19,9 @@ done
 
 >&2 echo "Postgres is up - continuing..."
 
+echo "Compiling message files"
```

Comment:
Should this happen here or in `csu update`, what is better for developers?

Reply:
As discussed offline, decided to leave here, because compiling message files requires a django reboot

Reply:
Agreed over phone this is suitable.